meta/main.yml

```
@@ -1,11 +1,16 @@
 ---
-dependencies: []
+dependencies:
+  - src: https://github.com/cisagov/ansible-role-docker
+    name: docker
```

I'm trying to alphabetize these YAML attributes going forward:
```suggestion
  - name: docker
    src: https://github.com/cisagov/ansible-role-docker
```